### PR TITLE
Replace `ind_active` for `active_cells` in pf simulations

### PR DESCRIPTION
**Pull request by santisoler**
#### Summary

Replace the `ind_active` argument in potential field simulation classes for `active_cells`. Add tests that check the corresponding warnings and errors are being raised. Update example, tutorials and tests so they use `active_cells` instead of `ind_active`.

#### PR Checklist
* [ ] If this is a work in progress PR, set as a Draft PR
* [ ] Linted my code according to the [style guides](https://docs.simpeg.xyz/content/getting_started/contributing/code-style.html).
* [ ] Added [tests](https://docs.simpeg.xyz/content/getting_started/practices.html#testing) to verify changes to the code.
* [ ] Added necessary documentation to any new functions/classes following the
      expect [style](https://docs.simpeg.xyz/content/getting_started/practices.html#documentation).
* [ ] Marked as ready for review (if this is was a draft PR), and converted
      to a Pull Request
* [ ] Tagged ``@simpeg/simpeg-developers`` when ready for review.

#### Reference issue

Part of #1121

### CODE REVIEW

**Comment by jcapriot** [comment]
Thinking a little forward here…

Probably best practice to actually leave `ind_active` as a keyword argument in `__init__` (but move it to the end). Then, leave it in the docstring (at the appropriate location), but mark it as deprecated in the parameter’s description.

I know it’s not how we’ve done it elsewhere, but I think we should start doing it that way.

File: simpeg/potential_fields/base.py [hunk 1]
```diff
@@ -92,9 +94,25 @@ def __init__(
         # If deprecated property set with kwargs
         if "actInd" in kwargs:
             raise AttributeError(
-                "actInd was removed in SimPEG 0.17.0, please use ind_active"
+                "actInd was removed in SimPEG 0.17.0, please use 'active_cells'"
```
Comment (jcapriot) [inline simpeg/potential_fields/base.py:97]: We can probably just remove this check now, been a few years since 0.17.0

**Comment by santisoler** [comment]
> Thinking a little forward here…
> 
> Probably best practice to actually leave `ind_active` as a keyword argument in `__init__` (but move it to the end). Then, leave it in the docstring (at the appropriate location), but mark it as deprecated in the parameter’s description.
> 
> I know it’s not how we’ve done it elsewhere, but I think we should start doing it that way.

Makes sense! I agree that we should start doing it that way.

**Comment by santisoler** [comment]
@jcapriot should we merge this one?

**Pull request merged**